plugins/reccobeats.py

```
@@ -0,0 +1,104 @@
+        seed_label = query
+
+    # Call ReccoBeats recommendation endpoint
+    session = get_session()
+    try:
+        r = session.get(
+            f"{API_BASE}/track/recommendation",
+            params={"seeds": track_id, "size": 5},
+            headers={"Accept": "application/json"},
+        )
+        r.raise_for_status()
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)